Code/GraphMol/RascalMCES/RascalOptions.h

```diff
@@ -58,6 +58,12 @@ struct RDKIT_RASCALMCES_EXPORT RascalOptions {
   bool ignoreAtomAromaticity = true; /* If true, atoms are matched just on
                                         atomic number; if false, will treat
                                         aromatic and aliphatic as different. */
+  uint minCliqueSize = 0; /* Normally, the minimum clique size is specified
```

Comment:
`uint` isn't a c++ type:
```suggestion
  unsigned int minCliqueSize = 0; /* Normally, the minimum clique size is specified
```